### PR TITLE
MAT-9774 updating spring boot starter parent

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADIE REST Commons PR

Jira Ticket: [MAT-9774](https://jira.cms.gov/browse/MAT-9774)
(Optional) Related Tickets:

### Summary
updating spring boot starter parent

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>3.4.10</version>
+        <version>3.5.13</version>
```
Comment (adongare) [inline pom.xml:8]: may update snapshot version as well so that new version would be published which then can be pulled into other services?

Comment (sb-prateekkeerthi) [reply pom.xml:8]: I just updated it thanks for the feedback

**Review by adongare — APPROVED** [review]
